extra/utils/validation.js

```diff
@@ -0,0 +1,93 @@
+let user=[
+    {
+        traineeEmail: 'trainee1@successive.tech',
+        reviewerEmail: 'reviewer1@successive.tech'
+    },
+    {
+        traineeEmail: 'aryan.singhal@successive.tech',
+        reviewerEmail: 'rahul.sadhukhan@successive.tech'
+    },
+    {
+        traineeEmail: 'trainee1@gmail.com',
+        reviewerEmail: 'reviewer1@hotmail.com'
+    },
+    {
+        traineeEmail: 'Aman@successive.tech',
+        reviewerEmail: 'vinay.chaudhary@successive.tech'
+    }
+];
+let regex=/[a-z]([[-]*\w+[.]*){1,63}@successive[.]tech$/gmi;
+let email;
+//let temp=[];
+let true_arr=[];
+let false_arr=[];
+validateUser(user);
```

Comment:
You have called this function here but you defined the function below, is this working and if yes, then why?

Reply:
Yes, it is working due to hoisting i.e we can use a variable before declaring.